Numpy/EyeandIdentity.py

```diff
@@ -6,6 +6,7 @@
 Created   : 15 July 2016
 Problem   : https://www.hackerrank.com/challenges/np-eye-and-identity/problem
 '''
-import numpy
-n,m = map(int,input().split())
-print(numpy.eye(n,m,k=0))
+import numpy as np
+np.set_printoptions(sign=' ')
+print(np.eye(*map(int, input().split())))
```

Comment:
As per the tutorial suggests we can alternatively set the print options to `1.13`.

```
np.set_printoptions(legacy='1.13')
n, m = map(int, input().split())
print(np.eye(n, m, k=0))
```

Reply:
Done. Changes made as per suggestion.